data/PDM_Message_Testing_Files/PDM_test_Speed.json

```diff
@@ -0,0 +1,66 @@
+{
+  "ode": {
+    "version": 2
+  },
+  "pdm": {
+    "timestamp": "2017-12-01T17:47:11-05:00",
+    "sampleStart": "200",
+    "sampleEnd": "220",
+    "heading": "1010101010101010",
+    "termTime": "400",
+    "snapshot": "SnapshotTime",
+    "sTime": {
+      "speed1": "15",
+      "speed2": "20",
+      "second1": "5",
+      "second2": "7"
+    },
+    "tsInterval": "50",
+    "dataList": [
+      {
+        "dataType": "5",
+        "subType": "7",
+        "sendOnLessThanValue": "-1500",
+        "sendOnMoreThanValue": "-1000",
+        "sendAll": "true"
+      }
+    ],
+    "params": {

```

Comment:
Wouldn't it be better if this is called rsuParams and get rid of all those rsuPDM prefixes for each rsuParma?